docs/organizer-guide/vm-organize-quorum-meetup.md

```diff
@@ -0,0 +1,76 @@
+
+The suggested timeframe for completing each of the preparation steps are:
+
+|  Step              |  Timeframe          |
+|  ---------------   |  ----------------   |
+|   Confirm details  |   1 month before    |
+|   Create event     |   3 weeks before    |
+|   Publicize event  |   2 weeks before    |
+
+## Planning the meetup
+
+To plan the meetup, begin by brainstorming possible speakers or presenters that you'd like to consider for your meetup. As you are brainstorming, ask yourself a few questions:
+
+- **Is there anyone in my immediate social network that would make a great speaker, such as a friend, co-worker, or a member of my local meetup?** You may be surprised to find good speakers in your network! Speakers don't have to be technical writers by profession. They can be developers, UX designers, product managers, Scrum masters, or other related professionals---they all have useful insights to share.
+- **Have I read an interesting book on a documentation-related topic that I'd like to hear more about?** Speakers are usually willing to promote and discuss their books, especially to receptive audiences. Most people you approach are likely to say yes, so go ahead and dream big!
+- **Have I heard or watched any interesting presentations or podcast episodes outside of Write the Docs that I'd like to hear more about?** Who seems to be on the speaking circuit right now? Sometimes you can find good speakers that have presented at conferences or other local meetups, such as the Society of Technical Communicators, Confab, or other technology or content-oriented conferences. Speakers who have already given or prepared a presentation are usually more than happy to give the same presentation again.
+- **Did someone in the WTD community creatively solve a problem or develop a novel approach to a documentation task?** Sometimes people in the Write the Docs community have really interesting approaches to solving documentation problems. Sharing those problems and cool solutions can help others in their own sticky situations. Feel free to reach out to members of our community who might be interested in providing more details.
+- **Does someone who works for a company that makes documentation tools want to speak to our audience?** While we don't want every presentation to turn into a product demo, members of our community might be genuinely interested in hearing about new products or tools that are being developed to help solve documentation problems. Some product demos can be quite interesting and these kind of speakers will (of course) jump at the opportunity to speak to our community.
+
+If you're stuck, ping into the #meetup-organizers-quorum channel on Slack to brainstorm possible speakers. 
+
+In the future, we hope to provide a speaker database to help assist with this part of the process. If you'd like to volunteer to help organize this database, we're happy to help you get started. 
+
```

Comment:
@barriebyron , we need to remove this last paragraph. At the time this was originally written, this was true. But now, due to privacy concerns, we no longer have plans to create a speaker database and I highly doubt there will be future plans to make a new one.